server/miljø.ts

```diff
@@ -56,31 +35,34 @@ const devMiljø: Environment = {
     'https://familie.ekstern.dev.nav.no/familie/alene-med-barn/soknad/barnetilsyn',
   søknadSkolepengerUrl:
     'https://familie.ekstern.dev.nav.no/familie/alene-med-barn/soknad/skolepenger',
-  infoSideOvergangsstønadUrl: 'https://www.ekstern.dev.nav.no/overgangsstonad-enslig',
-  infoSideBarnetilsynUrl: 'https://www.ekstern.dev.nav.no/barnetilsyn-enslig',
-  infoSideSkolepengerUrl: 'https://www.ekstern.dev.nav.no/skolepenger-enslig',
-  saksbehandlingstiderUrl: 'https://www.ekstern.dev.nav.no/saksbehandlingstider',
+  infoSideOvergangsstønadUrl: 'https://www.nav.no/overgangsstonad-enslig',
```

Comment:
Hvis vi skal ha "default" miljøvariabler så foretrekker jeg at alle sammen gjelder for prod-miljøet, så kan lokalt/dev overskrive det som evt. skal overskrives... Risikoen for å få ut en preprod-url i produksjonsmiljøet blir ganske høy når vi mikser det.

Reply:
Ja, jeg tenkte motsatt - at det var tryggere å defaulte til dev, samt at den var mest lik de andre miljøene.